homeassistant/components/lifx/light.py

```diff
@@ -212,6 +213,9 @@ async def set_state(self, **kwargs: Any) -> None:
                 return
 
             if ATTR_INFRARED in kwargs:
+                _LOGGER.warning(
+                    "The infrared attribute of the lifx.set_state service is deprecated and will be removed in a future version"
```

Comment:
What should they do instead?

Reply:
Set the value using the Infrared Brightness entity.

Reply:
Sorry for the lack of clarity. We should tell them that.